src/karmen_backend/openapi/swagger.yaml

```diff
@@ -0,0 +1,1544 @@
+              properties:
+                password:
+                  type: string
+                  format: password
+                new_password:
+                  type: string
+                  format: password
+                new_password_confirmation:
+                  type: string
+                  format: password
+      tags:
+        - users-me
+      responses:
+        '200':
+          description: 'Ok'
+        '401':
+          description: 'Unauthorized'
+
+  '/users/me/tokens':
+    get:
+      summary: 'List users API tokens'
+      operationId: server.routes.users_me.list_api_tokens
+      tags:
+        - users-me
+      responses:
+        '200':
+          description: 'List of tokens'
+    post:
+      summary: 'Create API token'
+      operationId: server.routes.users_me.create_api_token
+      tags:
+        - users-me
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                name:
+                  type: string
+      responses:
+        '200':
+          description: 'OK'
+  '/users/me/tokens/{jti}':
+    delete:
+      summary: 'Create API token'
+      operationId: server.routes.users_me.revoke_api_token
+      tags:
+        - users-me
+      parameters:
+        - $ref: '#/components/parameters/jti'
+      responses:
+        '204':
+          description: 'Revoked'
+
+
+
+  '/organizations/{org_uuid}/users':
+    post:
+      summary: Add user to organization.
+      description: If user exists, he is added to the org. If he does not, an account is created and he receives an email
+      operationId: server.routes.users.add_user_to_org
+      tags:
+        - users
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                email:
+                  type: string
+                  format: email
+                role:
+                  type: string
+                  enum: [user, admin]
+      responses:
+        '200':
+          description: ''
+        '400':
+          description: Unable to create or add user
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              examples:
+                Missing email:
+                  value:
+                    message: Missing email
+                Bad role:
+                  value:
+                    message: Bad role
+                Invalid email:
+                  value:
+                    message: Invalid email
+    get:
+      summary: 'Get list of users in organization'
+      operationId: server.routes.users.list_users
+      tags:
+        - users
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      responses:
+        '200':
+          description: 'List of users'
```

Comment:
Shape of the response is missing

Reply:
As the main goal of this task was to enforce API requests validation and it was already taking too long, I described only `/printers` responses, and the rest will be done later. 
We could discuss this at next meeting if you believe it should be done all at once.